res/controllers/Traktor-Kontrol-S3-hid-scripts.js

```diff
@@ -721,16 +721,12 @@ TraktorS3.Deck.prototype.wheelDeltas = function(value) {
 
     if (prevTime > timeval) {
         // We looped around.  Adjust current time so that subtraction works.
-        timeval += 0x100000;
+        timeval += 0x1000000;
```

Comment:
here is the timecode wraparound fix value.  Note that this doesn't protect against the user not touching the jog wheel for a large amount of time (enough for the internal counter to wrap around more than once)

Reply:
ok I added wallclock protection